File: scripts/fix_page.js
const fs = require('fs');

let content = fs.readFileSync('c:/Users/charl/OneDrive/Desktop/work/src/app/page.tsx', 'utf-8');

// Replace ternary class logic for tableDensity
content = content.replace(/\$\{tableDensity === 'fit' \? '[^']*' : '([^']*)'\}/g, "");

// Fix the shortCode logic
content = content.replace(/shortCode=\{tableDensity === 'fit'\}/g, "shortCode={false}");

// Remove the Fit Screen button logic completely
content = content.replace(/<button[^>]*>\s*Fit Screen\s*<\/button>/gm, "");

// Remove the divider line next to Fit Screen button
content = content.replace(/<div className="h-4 w-px bg-\[#333333\] mx-1"><\/div>/g, "");

fs.writeFileSync('c:/Users/charl/OneDrive/Desktop/work/src/app/page.tsx', content, 'utf-8');

File: scripts/fix_page.py
import re

with open('c:/Users/charl/OneDrive/Desktop/work/src/app/page.tsx', 'r', encoding='utf-8') as f:
    content = f.read()

# Replace ternary class logic for tableDensity
content = re.sub(r"\$\{tableDensity === 'fit' \? '[^']*' : '([^']*)'\}", r"\1", content)

# Fix the shortCode logic
content = re.sub(r"shortCode=\{tableDensity === 'fit'\}", r"shortCode={false}", content)

# Remove the Fit Screen button logic completely
button_pattern = re.compile(r"<button[^>]*>\s*Fit Screen\s*</button>", re.MULTILINE | re.DOTALL)
content = button_pattern.sub("", content)

# Remove the divider line next to Fit Screen button
divider_pattern = re.compile(r'<div className="h-4 w-px bg-\[\#333333\] mx-1"></div>')
content = divider_pattern.sub("", content)

with open('c:/Users/charl/OneDrive/Desktop/work/src/app/page.tsx', 'w', encoding='utf-8') as f:
    f.write(content)
